declaration/AnimationManager.d.ts

```diff
@@ -45,6 +45,6 @@ export declare class AnimationManager {
     animateTo(destPos: Axis, duration: number, option?: ChangeEventOption): void;
     private frame(param);
     easing(p: any): number;
-    setTo(pos: Axis, duration?: number): this;
+    setTo(pos: Axis, duration?: number, useCircular?: boolean): this;
```

Comment:
setTo is added `useCircular` parameter.
but setBy is not provided `userCircluar` parameter.

I think all of the circular param are available or not provided.

Reply:
@sculove I apply your review. and I choose not using circular option for simplicity. If axis has a `circular` option as `true` It will works as circular mode. 

eg. range = [0 ~ 4], circular = true , you setTo(5), it will be set to `1`. not `4`(BEFORE RESULT)

and I checked that flicking component works well by applying change.

But there may be some argue because it makes some change. Check plz @sculove @happyhj @netil